src/main/java/cn/nukkit/event/player/PlayerAsyncPreLoginEvent.java

```diff
@@ -23,7 +25,8 @@ public static HandlerList getHandlers() {
 
     private final String name;
     private final UUID uuid;
-    private final String xuid;
+    private final LoginChainData chainData;
```

Comment:
Shouldn't we have a getter for this too?

Reply:
Solved